cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,46 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+  url = 'article';
+
+  get titleOfArticle() {
+    return cy.get('h1');
+  }
+
+  get editButton() {
+    return cy.get('.container > .article-meta > ' +
+    ':nth-child(3) > .btn-outline-secondary');
+  }
+
+  get deleteButton() {
+    return cy.get('.container > .article-meta > ' +
+    ':nth-child(3) > .btn-outline-danger');
+  }
+
+  get articleDeleteOnProfilePage() {
+    return cy.get('h2');
+  }
+
+  assertArticleOnProfilePageIsDeleted() {
+    this.articleDeleteOnProfilePage.should('contain.text',
+      'This page could not be found');
```

Comment:
The text 'This page could not be found' should match exactly what is expected when an article is deleted. Ensure that this text is correct and update it if the application's message changes.